pkg/fieldpath/fieldpath_test.go

```diff
@@ -268,3 +284,60 @@ func TestSplitMaybeSubscriptedPath(t *testing.T) {
 		}
 	}
 }
+
+// TestFormatMap
+func TestFormatMap(t *testing.T) {
+	type args struct {
+		m map[string]string
+	}
+	tests := []struct {
+		name       string
+		args       args
+		wantFmtStr string
+	}{
+		{
+			name: "nil",
+			args: args{
+				m: nil,
+			},
+			wantFmtStr: "",
+		},
+		{
+			name: "label",
+			args: args{
+				m: map[string]string{
+					"beta.kubernetes.io/os":                 "linux",
+					"kubernetes.io/arch":                    "amd64",
+					"kubernetes.io/hostname":                "master01",
+					"kubernetes.io/os":                      "linux",
+					"node-role.kubernetes.io/control-plane": "true",
+					"node-role.kubernetes.io/master":        "true",
+				},
+			},
+			wantFmtStr: "beta.kubernetes.io/os=\"linux\"\nkubernetes.io/arch=\"amd64\"\nkubernetes.io/hostname=\"master01\"\nkubernetes.io/os=\"linux\"\nnode-role.kubernetes.io/control-plane=\"true\"\nnode-role.kubernetes.io/master=\"true\"",
```

Comment:
this is really difficult to evaluate for correctness... I'd suggest using ` quoted strings, like this:

```suggestion
			wantFmtStr: `beta.kubernetes.io/os="linux"
kubernetes.io/arch="amd64"
kubernetes.io/hostname="master01"
kubernetes.io/os="linux"
node-role.kubernetes.io/control-plane="true"
node-role.kubernetes.io/master="true"`,
```

same comment applies below

Reply:
Fixed